core/src/main/java/org/elasticsearch/rest/action/admin/cluster/storedscripts/RestPutStoredScriptAction.java

```diff
@@ -43,21 +43,6 @@ protected RestPutStoredScriptAction(Settings settings, RestController controller
         if (registerDefaultHandlers) {
             controller.registerHandler(POST, "/_scripts/{lang}/{id}", this);
             controller.registerHandler(PUT, "/_scripts/{lang}/{id}", this);
-
-            controller.registerHandler(PUT, "/_scripts/{lang}/{id}/_create", new CreateHandler(settings, controller, client));
```

Comment:
The fact that the create endpoints have been removed should be documented in the breaking changes for both scripts and templates.